README.md

```diff
@@ -25,6 +25,33 @@ are not required when you don't want to use the WordPress image upload with S3.
 
 In reverse, if you activate the already preinstalled S3 plugin you must specify these variables.
 
+By default, the default AWS endpoint (s3.amazonaws.com) will be used.
+Be careful of that if you're using another provider.
```

Comment:
nitpick: I don't see what information this sentence adds. This is already explained in the next section
```suggestion
```